### DISCUSSION TITLE
raw.github.com has wrong certificate

### DISCUSSION THREAD

**Discussion by kyle999312** [community/community#41854] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Looks like raw.github.com has a cert for github.io instead of github.com.  So trying to get artifacts fail
![image](https://user-images.githubusercontent.com/16343167/207933718-4c9bed4a-e0e2-4ca9-95cd-ec707384013a.png)

**Comment by ms20de** [comment]
Same problem as here:

https://github.com/orgs/community/discussions/41838
